packages/commonwealth/client/scripts/views/pages/discussions_redirect.tsx

```diff
@@ -39,7 +46,7 @@ export default function DiscussionsRedirect() {
       default:
         navigate('/discussions', dontAddHistory);
     }
-  }, [navigate]);
+  }, [navigate, isRedirected, app.chain, routerParams.scope]);
```

Comment:
Fix code quality check

Reply:
tested it without app.chain, and still works, so removed it.